telemetry.go

```diff
@@ -0,0 +1,31 @@
+package gocron
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	jobLatency = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name: "gocron_job_latency_seconds",
+		Help: "The latency of jobs in seconds.",
+	}, []string{"job_name"})
+	// jobErrors = prometheus.NewCounter(prometheus.CounterOpts{
+	// 	Name: "gocron_jobs_failures_total",
+	// 	Help: "The total number of job failures.",
+	// })
+
+	metricsEnabled = false
+)
+
+func init() {
+	prometheus.MustRegister(jobLatency)
+	// prometheus.MustRegister(jobErrors)
+}
```

Comment:
this should check if metrics are enabled before registering the metric, and if we can't do that within the init, function, then move this inside the EnableMetrics func